src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
```

Comment:
Consider renaming firstLeg to legA for clarity and consistency with mathematical notation 📚